data.md

```diff
@@ -101,17 +101,20 @@ These can be used for pattern-matching on the LHS of rules as well (`macro` attr
                   | #rangeSFixed  ( Int , Int , Int )
                   | #rangeUFixed  ( Int , Int , Int )
                   | #rangeAddress ( Int )
-                  | #rangeBytes   ( Int , Int )
+                  | #rangeBytes   ( Int , Int )             [function]
```

Comment:
Formating. Extend the `---` underneath to the end of the line. No need for spaces before `[function]`. See the rest of the semantics for formatting guide.